docs/gettingstarted/index.md

```diff
@@ -1,60 +1,119 @@
+    * **Visual Studio Code**: Run the `getVSCodeRuns` task.
+    * **Other IDEs**: You can run the configurations directly using `gradle run*` (e.g., `runClient`, `runServer`, `runData`, `runGameTestServer`). These can also be used with the supported IDEs.
 
 Customizing Your Mod Information
 --------------------------------
 
-Edit the `build.gradle` file to customize how your mod is built (the file names, versions, and other things). Most things in the `build.gradle` can be removed or customized there.
+Edit the `build.gradle` file to customize how your mod is built (e.g., file name, artifact version, etc.).
 
 !!! important
-    **Do not** edit the `settings.gradle` unless you know what you are doing. The default text is necessary to get the ForgeGradle plugin.    
+    Do **not** edit the `settings.gradle` unless you know what you are doing. The file specifies the repository that [ForgeGradle] is uploaded to.
+
+### Recommended `build.gradle` Customizations
 
-### Simple `build.gradle` Customizations
+#### Mod Id Replacement
 
-These customizations are highly recommended for all projects.
+Replace all occurrences of `examplemod`, including [`mods.toml` and the main mod file][modfiles] with the mod id of your mod. This also includes changing the name of the file you build by setting `archivesBaseName` (this is typically set to your mod id).
 
-* To change the name of the file you build - edit the value of `archivesBaseName` to suit.
-* To change your "maven coordinates" - edit the value of `group` as well.
-* To change the version number - edit the value of `version`.
-* To update the run configurations - replace all occurrences of `examplemod` to the mod id of your mod.
+```gradle
+// In some build.gradle
+archivesBaseName = 'mymod'
+```
 
-### Migration to Mojang's Official Mappings
+#### Group Id
 
-Forge uses Mojang's Official Mappings, or MojMaps, for the forseeable future. The official mappings provide class, method, and field names. Parameters and javadocs are not provided by this mapping set. Currently, there is no guarantee that these mappings are legally safe; however, Forge has decided to adopt them in good faith since Mojang wants them to be used. You can read about [Forge's stance here][mojmap].
+The `group` property should be set to your [top-level package][packaging], which should either be a domain you own or your email address:
+
+Type      | Value             | Top-Level Package
+:---:     | :---:             | :---
+Domain    | example.com       | `com.example`
+Subdomain | example.github.io | `io.github.example`
+Email     | example@gmail.com | `com.gmail.example`
+
+```gradle
+// In some build.gradle
+group = 'com.example'
+```
+
+The packages within your java source (`src/main/java`) should also now conform to this structure, with an inner package representing the mod id:
+
+```text
+com
```

Comment:
```suggestion
com (TLD) 
```

Reply:
I think this should go on the `example` section where it says `(top-level package (group))`